src/fsharp/vs/service.fs

```diff
@@ -1372,153 +1413,120 @@ module internal Parser =
          
 
     /// Error handler for parsing & type checking while processing a single file
-    type ErrorHandler(reportErrors, mainInputFileName, tcConfig: TcConfig, source: string) =
-        let mutable tcConfig = tcConfig
+    type ErrorHandler(reportErrors, mainInputFileName, errorSeverityOptions: FSharpErrorSeverityOptions, source) =
+        let mutable options = errorSeverityOptions
         let errorsAndWarningsCollector = new ResizeArray<_>()
         let mutable errorCount = 0
-         
+
         // We'll need number of lines for adjusting error messages at EOF
         let fileInfo = GetFileInfoForLastLineErrors source
-         
+
         // This function gets called whenever an error happens during parsing or checking
-        let diagnosticSink sev (exn:PhasedDiagnostic) = 
```

Comment:
Try to avoid the whitespace changes in a PR.  If you'd like to remove all trailing whitespaces, then please submit a single PR tht does it globally across all source files under ``src`` and ``vsintegration\src`` - that would be great - thanks!

Reply:
I would be easier to do it in a few parts, I think, rather than a single PR.
Should I revert these formatting changes in this PR?

Reply:
@auduchinok it's not a big deal for this one - you can leave these changes in - it was just a comment for future PRs